aws/sdk/integration-tests/dynamodb/tests/movies.rs

```diff
@@ -125,8 +125,9 @@ where
         &self,
         response: Result<&SdkSuccess<DescribeTableOutput>, &SdkError<DescribeTableError>>,
     ) -> RetryKind {
-        match self.inner.classify(response.clone()) {
+        match self.inner.classify(response) {
```

Comment:
we have a few other classify implementations (imds at least) that need the same treatment

Reply:
I'm worried about this breaking change silently breaking customers' customer retry behavior, so I'm going to rename `NotRetryable` to something else to force a compilation error so that they revisit their retry classification implementations. This should also have the side effect of revealing all of the broken classify implementations in our own code. It looks like there are a few of them in our credential providers chain.